include/rdma/fi_ext.h

```diff
@@ -155,6 +155,54 @@ struct fi_peer_cq_context {
 };
 
 
+/* Peer shared rx context */
+struct fid_peer_srx;
+
+/* Castable to dlist_entry */
+struct fi_peer_rx_entry {
+	struct fi_peer_rx_entry *next;
+	struct fi_peer_rx_entry *prev;
+	fi_addr_t addr;
```

Comment:
Which address should it be?
The address in peer's AV or the address in owner's AV.

Reply:
Address is relative to the caller (peer provider).  That's all the information that it has.